### PR TITLE
Library: document new text filter operator '=' (find exact matches)

### PR DESCRIPTION
**Pull request by ronso0**
document https://github.com/mixxxdj/mixxx/pull/12063

https://deploy-preview-590--mixxx-manual.netlify.app/chapters/library#using-search-operators

### CODE REVIEW

File: source/chapters/library.rst [hunk 1]
```diff
@@ -413,6 +413,19 @@ Mixxx supports the following filters:
      It doesn't matter if you have space between the colon and the argument
      or not. Quotes must be used for multi-word text arguments.
 
+  You can use the **=** operator to find only tracks where the entire property
+  text equals the search term. You can also enclose multiple terms in quotes.
+  The first of the examples below will find only tracks where the title is
+  "wow" (case-insensitive like the default search), i.e. not "wow!", "wowy"
+  or "wow (Hej Remix)".
```
Comment (Swiftb0y) [inline source/chapters/library.rst:420]: I think there should also be an example with a prefix, eg: 
```suggestion
  "wow" (case-insensitive like the default search), i.e. not "wowy", "oh wow!"
  or "wow (Hej Remix)".
```

File: source/chapters/library.rst [hunk 2]
```diff
@@ -413,6 +413,19 @@ Mixxx supports the following filters:
      It doesn't matter if you have space between the colon and the argument
      or not. Quotes must be used for multi-word text arguments.
 
+  You can use the **=** operator to find only tracks where the entire property
+  text equals the search term. You can also enclose multiple terms in quotes.
```
Comment (Swiftb0y) [inline source/chapters/library.rst:417]: I think this is clearer.
```suggestion
  You can use the **=** operator to find only tracks where the entire property
  text equals the search term. As without **=**, you must use quotes for more than one word.
```

**Review by Swiftb0y — APPROVED** [review]
LGTM. thank you for cleaning up the loose ends. Any1 else wanting to review?

**Comment by daschuer** [comment]
I am missing something like the PR headline in the text.

**Comment by ronso0** [comment]
What exactly do you have in mind?

IMHO this is clarified with
> You can use the = operator to find only tracks where the entire property text equals the search term.

Note that this is just an extension, not an entirely new feature which would deserve more emphasis.

File: source/chapters/library.rst [hunk 3]
```diff
@@ -413,6 +413,19 @@ Mixxx supports the following filters:
      It doesn't matter if you have space between the colon and the argument
      or not. Quotes must be used for multi-word text arguments.
 
+  You can use the **=** operator to find only tracks where the entire property
```
Comment (daschuer) [inline source/chapters/library.rst:416]:
```suggestion
  You can use **=** for finding exact matches. It reveals only tracks where the entire property
```

**Comment by ronso0** [comment]
Alright, done.

**Review by daschuer — APPROVED** [review]
Thank you.

**Pull request merged**